src/style.css

```diff
@@ -0,0 +1,125 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url("/fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("/fonts/Avenir-Heavy.ttf");
+}
+
+.form {
+  display: block;
+  box-sizing: border-box;
+  width: 100%;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-weight: 300;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.form__big {
+  height: 70px;
+  border-radius: 5px;
+}
+
+.form__small {
+  height: 42px;
+  border-radius: 4px;
+}
+
+.search {
+  box-sizing: border-box;
+  position: absolute;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border: none;
+  outline: none;
+}
+
+.search__big {
+  padding-left: 62px;
+}
+
+.search__small {
+  padding-left: 33px;
+}
+
+.search__big::placeholder {
+  width: 129px;
+  height: 22px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  position: absolute;
+  left: 62px;
+  top: 36%;
+  bottom: 37%;
+  color: #3d4e61;
+}
+
+.search__small::placeholder {
```

Comment:
not fixed
DRY
you don't need classes like `__big` and `__smal`, apply common styles for search and then change just styles that differ
follow the checklist example
<img width="937" alt="image" src="https://user-images.githubusercontent.com/102411130/234775582-f5097dfe-0bd9-474c-809a-2df82e5b6a8e.png">